Documentation/operations/upgrade.rst

```diff
@@ -440,6 +440,8 @@ Removed Options
 * The long defunct and undocumented ``single-cluster-route`` flag has been removed.
 
 * Deprecated options ``enable-k8s-event-handover`` and ``cnp-status-update-interval`` has been removed.
+* Deprecated values ``strict``, ``partial``, ``probe`` and ``disabled`` for ``kube-proxy-replacement`` flag have been
+  removed. Please use ``true`` and ``false`` instead.
```

Comment:
I would maybe summarise the differences between the option sets (if I remember: `true` == `strict`, `false` is like `disabled` but you can turn on individual features yourself) or at least link to the relevant docs (changed in https://github.com/cilium/cilium/commit/1a4e7d442543b5b09756c5ef2d0a5e68393f1020).

(Same for Helm.)

Reply:
+1

Reply:
Good point, I have added the reference pointing to KPR page :+1: